src/Features/Core/Portable/xlf/FeaturesResources.de.xlf

```diff
@@ -4,7 +4,7 @@
     <body>
       <trans-unit id="Add_parameter_to_0_and_overrides_implementations">
         <source>Add parameter to '{0}' (and overrides/implementations)</source>
-        <target state="new">Add parameter to '{0}' (and overrides/implementations)</target>
+        <target state="translated">Parameter zu "{0}" (und Außerkraftsetzungen/Implementierungen) hinzufügen</target>
```

Comment:
"Außerkraftsetzungen" is inconsistent with the rest of the German translation. In other places, "Überschreibungen" is used as the translation for the noun "overrides" (cf. line [1785 in `FeaturesResources.de.xlf`](https://github.com/dotnet/roslyn/blob/master/src/Features/Core/Portable/xlf/FeaturesResources.de.xlf#L1785), line [1477 in `CSharpResources.de.xlf`](https://github.com/dotnet/roslyn/blob/master/src/Compilers/CSharp/Portable/xlf/CSharpResources.de.xlf#L1477)).